### PR TITLE
Add missing fields from AutoSys

### PR DESCRIPTION
**Pull request by arusi-astro**
Adds missing fields that were found in our AutoSys dump file.

### CODE REVIEW

**Review by fritz-astronomer — APPROVED** [review]
Also - considering just taking any field we find... Would be riskier(?), but less maintenance and more chance of correctness over time. 

Thoughts, @arusi-astro ?

File: jilutil/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import os
 from datetime import datetime
 
-from jil_parser import JilParser
+from .jil_parser import JilParser
```
Comment (fritz-astronomer) [inline jilutil/__init__.py:9]: had changed this intentionally from the fork - does the relative import help? It was giving me issues occasionally

File: jilutil/auto_sys_job.py [hunk 2]
```diff
@@ -11,44 +11,6 @@ class AutoSysJob(UserDict):
...
-        'job_terminator': '',
-        'max_exit_status': '',
-        'max_run_alarm': '',
-        'min_run_alarm': '',
-        'n_retrys': '',
-        'priority': '',
-        'profile': '',
-        'run_window': '',
-        'term_run_time': ''
-    }
```
Comment (fritz-astronomer) [inline jilutil/auto_sys_job.py:50]: and, fwiw, I think these were needed to do the CSV headers correctly - but if we do want to keep that functionality, we can just generate a list of all the keys by reducing the list down

**Review by fritz-astronomer — APPROVED** [review]

File: jilutil/__init__.py (hunk 1 above)
Comment (arusi-astro) [reply jilutil/__init__.py:9]: We ve been running local development Orbiter with `python -m`: `python -m orbiter translate workflow/ output/ --ruleset override.translation_ruleset`.

This was the only way to get it to detect the `jil_parser` module, but maybe our way of calling Orbiter is wrong?

File: jilutil/auto_sys_job.py (hunk 2 above)
Comment (arusi-astro) [reply jilutil/auto_sys_job.py:50]: Yup, I noticed that once I ran the parser for the first time :facepalm: My bad

File: jilutil/__init__.py (hunk 1 above)
Comment (fritz-astronomer) [reply jilutil/__init__.py:9]: 🤷

**Review by fritz-astronomer — APPROVED** [review]

**Comment by arusi-astro** [comment]
Closed due to not being necessary to the parser.

**Pull request closed without merging**